lms/static/sass/views/_program-marketing-page.scss

```diff
@@ -0,0 +1,1696 @@
+                    @media (min-width: 1024px) {
+                        display: inline-block;
+                    }
+                }
+
+                + a {
+                    @media (max-width: 767px) {
+                        color: palette(primary, dark);
+                    }
+                }
+
+                &:first-child {
+                    color: palette(primary, dark);
+                    margin-bottom: 5px;
+                    @media (max-width: 767px) {
+                        margin-bottom: 0;
+                    }
+                    font-size: .85em;
+                    @media (max-width: 767px) {
+                        text-transform: uppercase;
+                    }
+                    @media (min-width: 768px) {
+                        width: 48%;
+                        color: palette(primary, dark);
+                        margin-bottom: 0;
+                        font-size: 1.024em;
+                    }
+                }
+            }
+        }
+    }
+    .data-table{
+        @media(max-width: 767px){
+            padding: 0 20px;
+        }
+        .list-divided{
+            @media(min-width: 767px){
+                padding: 0 12px !important;
+            }
+
+            .item{
+                margin-top: 0.625rem;
+                line-height: 1.4em;
+                padding: 15px 15px 19px 25px !important;
+                border-bottom: 1px solid $m-gray-l3 !important;
+                @media (min-width: 768px) {
+                    padding: 15px 0px 19px 0 !important;
+                }
+                @media(max-width: 767px){
+                    margin: 0 !important;
+                }
+                &:last-child{
+                    border-bottom: 0 !important;
+                }
+            }
+        }
+    }
+
+    .faq-links-list {
+
+        li {
+            margin-bottom: 15px;
+
+            a {
+                display: block;
+                position: relative;
+                color: $blue;
+                padding-left: 30px;
+
+                &:before {
+                    font-family: FontAwesome;
+                    content: "\f105";
+                    position: absolute;
+                    left: 5px;
+                    top: 1px;
+                }
+                & + div {
+                    padding: 10px 30px;
+                    font-size: 1rem;
+                    color: $black;
+                }
+            }
+            &.expanded {
+               a:before {
+                 content: "\f107";
+               }
+            }
+        }
+
+    }
+
+    .ui-state-default .ui-icon {
+        display: none;
+    }
+
+    //Modals
+    .modal {
+      border-radius: 0px !important;
+      background: #FFFFFF !important;
+
+      .inner-wrapper {
+        padding: 0px !important;
+        border: none !important;
+      }
+    }
+    .modal-custom {
+      width: 320px;
+      background-color: white;
```

Comment:
I've seen `$white` being used as well. Should this be a variable instead, too? Same goes for all the non-variable color attributes here.